ddtrace/propagation/http.py

```diff
@@ -1147,7 +1137,14 @@ def my_controller(url, headers):
                     context = HTTPPropagator._resolve_contexts(contexts, styles_w_ctx, normalized_headers)
                     if config.propagation_http_baggage_enabled is True:
                         _attach_baggage_to_context(normalized_headers, context)
-                    return context
+
+            # baggage
+            if config._propagation_style_extract and _PROPAGATION_STYLE_BAGGAGE in config._propagation_style_extract:
```

Comment:
Is there ever an instance when we won't have a value for `config._propagation_style_extract`? It looks like the rest of the method trusts that at the very least it's an empty list. So I think we can drop that check.

Reply:
Yeah, I originally didn't have the check for it, but Pylance was getting really angry at me saying that `config._propagation_style_extract` could be None.